### DISCUSSION TITLE
Could i get data from redis using NectJS

### DISCUSSION THREAD

**Discussion by HosMercury** [maxcountryman/tower-sessions#190] [Q&A] [answered]
I see data in redis like this 
`
��l��GFң�������user��created_at� 2024-03-29T22:47:01.496659+02:00�deleted_at��email��id�last_sign� 2024-03-29T22:47:01.496659+02:00�name�Ferris�provider�web�updated_at��username�ferris���a3�H�
`

is it encrypted anyway?
is it coupled with axum and rust ? or could I handle them from another server like Next?

**Comment by maxcountryman** [comment, marked as answer]
If you're using the RedisStore provided by redis-store then it's encoded by Rust MsgPack.

If you implement your own store you don't need to encode it this way. Alternatively you can decode the session manually.

Please be aware that tower-sessions assumes that only it has write access to session storage so if other systems mutate this state it may cause trouble.
